### DISCUSSION TITLE
Absence of preview text feature

### DISCUSSION THREAD

**Discussion by JacksonCampbell99** [community/community#5756] [Mobile]
The biggest thing I do on GitHub is previewing my Markdown text to check the formatting and for errors. I rely very heavily on that feature for new posts and comments. However, that feature is only available on the website and not on iOS or Android. I would love to be able to solely use the app on mobile for a smooth and convenient experience, but I will not be able to contribute without that ability on the mobile application; I will only be able to keep up to date with things and do simple comments. I would love to see the preview text functionality added to the apps.

**Comment by alcere** [comment]
Hi! thanks for your feedback, we are sharing it with the team to be evaluated, thanks!

**Reply by jsoref** [reply]
Does this the request cover the inability to preview comments/submission for Issues and Discussions?

Ostensibly, I have no idea how this comment will appear, although, my guess is it'll have a picture that looks like the pictured drafted entry which in turn has text that refers to a picture that I also couldn't see/verify before posting, which is pretty awkward:

![Screenshot_20230111-092714.png](https://user-images.githubusercontent.com/2119212/211832058-5e316e2e-ddbb-4cf4-a002-161b4e06f1e8.png)

**Reply by JacksonCampbell99** [reply]
Yes, that would be good. Same thing. Or alternatively for that just show the picture without showing the markdown on the app.

**Reply by DarkCrypt** [reply]
It's absolutely insane that this necessary feature still doesn't exist. An annoyance that should of been fixed long ago.
